plugins/ilert/src/components/AlertsPage/AlertsTable.tsx

```diff
@@ -140,14 +140,19 @@ export const IncidentsTable = ({
       </Typography>
     ),
   };
-  const assignedToColumn: TableColumn = {
-    title: 'Assigned to',
-    field: 'assignedTo',
+  const respondersColumn: TableColumn = {
+    title: 'Responders',
+    field: 'responders',
     cellStyle: !compact ? mdColumnStyle : lgColumnStyle,
     headerStyle: !compact ? mdColumnStyle : lgColumnStyle,
     render: rowData => (
-      <Typography noWrap>
-        {ilertApi.getUserInitials((rowData as Incident).assignedTo)}
+      <Typography>
+        {(rowData as Alert).responders.map((value, i, arr) => {
```

Comment:
The responders themselves are displayed with their full name. Unfortunately i can't provide a link to a user page as it's an admin page only even if it would've provided more functionality to the list. 

I can't replicate a warning on the key property, as it's as you said only a concatenated string in the end. I don't know how `.join(', ')` would help me achieve the wanted behaviour better, correct me if i'm wrong.

Reply:
It's an array of strings though, not just a string. But maybe React is smart enough to not need keys if the things in there are all strings! But alright, let's leave this one as-is.